openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -423,9 +435,13 @@ OpenAssessment.ResponseView.prototype = {
     moveToNextStep: function() {
         var baseView = this.baseView;
         var usageID = baseView.getUsageID();
+        var view = this;
+
         this.load(usageID);
         baseView.loadAssessmentModules(usageID);
 
+        view.announceStatus = true;
```

Comment:
This is on submit.  Then when the page refreshes and renders from the submit it will announce the status.